docs/migration-guides/4.0.md

```diff
@@ -11,11 +11,11 @@ this migration guide covers the most notable changes.
 
 A couple of the properties in rules have changed:
 
-1. `enabled` was replaced with `recommended`
+1.) `enabled` was replaced with `recommended`
```

Comment:
why add `)`?

Reply:
Formatting was broken. It was showing up as a bullet point of 1 every time.

Reply:
Numbers should be intended like it was showing up. It looks weird like this.

Reply:
Instead of showing:

  1. 

  2. 

  3. 

It's showing 

  1. 

  1. 

  1. 

Studio seems to handle this specific markdown situation a little differently than GitHub.

Reply:
This is in studio - it it in Github that formatting is broken?

<img width="1250" alt="Screen Shot 2019-08-08 at 1 40 58 AM" src="https://user-images.githubusercontent.com/847542/62680756-98f8bf00-b97d-11e9-9770-592aa53031c8.png">

Reply:
Nah this was fixed recently: https://github.com/stoplightio/markdown-viewer/pull/20

Reply:
Now that the fix is in I've reverted this change.